lib/client/rpc/proposals.ts

```diff
@@ -169,26 +225,26 @@ export class FutarchyRPCProposalsClient implements FutarchyProposalsClient {
       })
       .preInstructions([
         createAssociatedTokenAccountIdempotentInstruction(
```

Comment:
Future note: We should fetch account if exists if not add these instructions.

This is probably what we want, but then we have to check if it exists..... 
https://solana-labs.github.io/solana-program-library/token/js/functions/getAssociatedTokenAddressSync.html

Will have to think on it some more.

This MAY work but seems to do an RPC call and create the account if not exists
https://solana-labs.github.io/solana-program-library/token/js/functions/getOrCreateAssociatedTokenAccount.html